Library/ViewModels/ChangeEmailViewModel.swift

```diff
@@ -101,7 +101,10 @@ public final class ChangeEmailViewModel: ChangeEmailViewModelType, ChangeEmailVi
     let isEmailVerified = userEmailEvent.values().map { $0.me.isEmailVerified }.skipNil()
     let isEmailDeliverable = userEmailEvent.values().map { $0.me.isDeliverable }.skipNil()
     let emailVerifiedAndDeliverable = Signal.combineLatest(isEmailVerified, isEmailDeliverable)
-      .map { $0 && $1 }
+      .map { isEmailVerified, isEmailDeliverable -> Bool in
+        let r = isEmailVerified && isEmailDeliverable
+        return r
```

Comment:
These changes are kind've unfortunate and seem unnecessary but I wasn't able to get this code to compile without introducing some of these local scope vars.